tubesync/common/templates/base.html

```diff
@@ -8,46 +8,82 @@
         opacity: 0;
       }
     </style>
-    <link type="text/css" rel="stylesheet" href="{% sass_src 'styles/tubesync.scss' %}" media="screen,projection"/>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@1.0.4/css/bulma.min.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.7.2/css/all.min.css" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link type="text/css" rel="stylesheet" href="{% static 'styles/tubesync.css' %}" media="screen,projection"/>
     <link rel="icon" href="{% static 'images/favicon.ico' %}" sizes="21x21" type="image/x-icon">
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <title>TubeSync - {% block headtitle %}Synchronize YouTube to your local media server{% endblock %}</title>
+    <script>
```

Comment:
This is the first of a few JS blocks - I think this is the first script in TubeSync.

I usually prefer implementing the script in TypeScript, but that introduces a bunch of extra build changes. Holding off until there's some consensus on what this should look like.

This script runs on page load to check what the stored theme is. This lives up here to prevent a flash of light mode styles before dark mode is applied.